gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -142,12 +150,13 @@ async fn pay_valid_invoice(
 ) -> anyhow::Result<()> {
     // User client pays test invoice
     let user_lightning_module = &user_client.get_first_module::<LightningClientModule>();
+    let gateway = user_lightning_module.select_active_gateway().await.ok();
     let OutgoingLightningPayment {
         payment_type,
         contract_id,
         fee: _,
     } = user_lightning_module
-        .pay_bolt11_invoice(invoice.clone(), ())
+        .pay_bolt11_invoice(gateway, invoice.clone(), ())
```

Comment:
why not call `pay_invoice` here?

Reply:
I think I got tired of repeating the same code after a while, introduced the fn but forgot that one.